internal/common/request.go

```diff
@@ -17,4 +18,9 @@ type Request struct {
 	Instance     *ssp.SSP
 	Logger       logr.Logger
 	VersionCache VersionCache
+	TopologyMode osconfv1.TopologyMode
+}
+
+func (r *Request) IsSingleModeTopology() bool {
```

Comment:
A typo. Should be `IsSingleNodeTopology`.

Reply:
LOL, I was thinking of SingleReplica and TopologyMode, and combined them in the most confusing way :)